src/OrganisationUnitTree/Label.js

```diff
@@ -0,0 +1,336 @@
+import { Checkbox, colors } from '@dhis2/ui-core'
+import React from 'react'
+import propTypes from 'prop-types'
+import cx from 'classnames'
+
+import { FolderOpen } from './icons/FolderOpen'
+import { FolderClosed } from './icons/FolderClosed'
+import { Single } from './icons/Single'
+import { Empty } from './icons/Empty'
+import { orgUnitPathPropType } from './propTypes'
+
+/**
+ * @param {Object} props
+ * @param {string} props.label
+ * @param {Function} props.onToggleOpen
+ * @param {bool} [props.loading]
+ * @returns {React.Component}
+ */
+const DisabledSelectionLabel = ({ label, loading, onToggleOpen }) => (
+    <SingleSelectionLabel
+        checked={false}
+        loading={loading}
+        label={label}
+        onChange={onToggleOpen}
+    />
+)
+
+DisabledSelectionLabel.propTypes = {
+    label: propTypes.string.isRequired,
+    onToggleOpen: propTypes.func.isRequired,
+    loading: propTypes.bool,
+}
+
+/**
+ * @param {Object} props
+ * @param {string} props.label
+ * @param {bool} [props.checked]
+ * @param {bool} [props.loading]
+ * @param {Function} [props.onChange]
+ * @returns {React.Component}
+ */
+const SingleSelectionLabel = ({ checked, label, onChange, loading }) => (
+    <span
+        onClick={event => {
+            const payload = { checked: !checked }
+            onChange(payload, event)
+        }}
+        className={cx({ checked, loading })}
+    >
+        {label}
+
+        <style jsx>{`
+            span {
+                display: inline-block;
+                background: transparent;
+                border-radius: 3px;
+                padding: 0 5px;
+                line-height: 26px;
+                white-space: nowrap;
+                cursor: pointer;
+                user-select: none;
+            }
```

Comment:
Let's style this more like the default `dense` node. Add: `font-size:14px;` and adjust line-height to `24px`. 

I notice that no color is set, presumably we're relying on the app to use the global theme. I think let's be safe and include `color: ${colors.grey900}`

Reply:
Done